src/Service/ZaakTypeService.php

```diff
@@ -456,6 +456,7 @@ public function caseTypeToZaakType(array $caseType, bool $flush = true)
 
         // Flush here if we are only mapping one zaaktype and not loopin through more in a parent function.
         if ($flush === true) {
+            $this->entityManager->flush();
             $this->entityManager->flush();
```

Comment:
Flush twice?

Reply:
Yup, somehow the self url is not flushed to the database if flush is triggered once, same holds for running some subscribers.